Source/Core/DolphinWX/Debugger/CodeView.cpp

```diff
@@ -366,6 +415,8 @@ void CCodeView::OnMouseUpR(wxMouseEvent& event)
   menu.AppendSeparator();
 #endif
   menu.Append(IDM_RENAMESYMBOL, _("Rename &symbol"))->Enable(isSymbol);
+  menu.Append(IDM_SETSYMBOLSIZE, _("&Set symbol size"))->Enable(isSymbol);
```

Comment:
Of course it's an issue. An accelerator is supposed to make jumping to desired actions easier, this effectively defeats the point of them if you can't immediately jump to what you want.

It's also a really good way to introduce random behavior and is also [advised against](https://msdn.microsoft.com/en-us/library/5967h0k4.aspx) in MS' documentation about accelerator tables.